dev_docs/cxg.md

```diff
@@ -119,46 +116,51 @@ This optional field contains a copy of the category color table, which MAY be us
 
 ## Corpora Schema Encoding
 
-The [Corpora schema](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md) and [Corpora AnnData encoding](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) define a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following shall apply.
+Note: This section is technically not part of the CXG specification, but rather provides an expectation of what metadata will be available in a CXG file that is converted from Corpora Schema-compliant H5AD file; we use "WILL" rather than "MUST" to document these expectations.
+
+The [Corpora Schema 2.0.0](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema) defines a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following WILL apply.   
 
 ### Corpora metadata property
 
-A CXG containing a Corpora dataset will contain a property in the __cxg_group_metadata__ field named `corpora`. The value will be a JSON encoded string, which in turn contains all properties defined in the [Corpora AnnData uns](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md#uns) container. For example:
+A CXG containing a Corpora dataset WILL contain a property in the `__cxg_group_metadata__` field named `corpora`. The value WILL be a JSON encoded string, which in turn contains all properties defined in the [Corpora Schema 2.0.0 "uns"](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/schema.md#uns-dataset-metadata) AnnData container. The entire encoding WILL be JSON, rather than a hybrid Python/JSON encoding, but will otherwise follow the data structure defined by the Corpora schema.
+
 
+These Corpora metadata properties WILL exist in the CXG `__cxg_group_metadata__` field named `corpora`: 
+* `schema_version`
+* `title`
+* `X_normalization`
+* `X_approximate_distribution`
+
+For example:
 ```
 {
     "corpora": {
-        "version": {
-            "corpora_schema_version": "1.0.0",
-            "corpora_encoding_version": "0.1.0",
-        }
+        "schema_version": "2.0.0",
+        "title": "Analysis of 8 datasets of healthy and diseased mouse heart",
+        "X_normalization": "CPM",
+        "X_approximate_distribution": "normal
```

Comment:
missing terminating quote

Reply:
purely to ensure reviewers are doing their job. 😉  fixed!